strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -101,7 +104,7 @@
 
     @XmlElement(name = "repository")
     @XmlElementWrapper(name = "group")
-    private Set<String> groupRepositories = new LinkedHashSet<>();
+    private List<String> groupRepositories = SetUniqueList.decorate(new LinkedList());
```

Comment:
@sbespalov , any objections to `org.apache.commons.collections.list.SetUniqueList` ?

Reply:
@fuss86 well this will not protect from storing duplicate list elements in OrientDB. Also I think that this will be replaced with some OrientDB internal `List` type while reading `Repository` form DB and following will produce duplicates:
```
Repostiory r = readFromDb(id);
r.getGroupRepositories().add("dupRepoElement");
r.getGroupRepositories().add("dupRepoElement");
```
Also, maybe I'm fixated on RDBMS, but are you sure that OrientDB will keep order for `List` elements while storing and then reading it back?

Reply:
Thanks. Unfortunately, this needs to be further improved and tested.

Reply:
Changed this to map. See new tests